cmd/gke-gcloud-auth-plugin/cred.go

```diff
@@ -127,8 +133,11 @@ func main() {
 		}
 	} else {
 		tokenProvider = &gcloudTokenProvider{
-			readGcloudConfigRaw: readGcloudConfigRaw,
-			readFile:            readFile,
+			readGcloudConfigRaw:       readGcloudConfigRaw,
```

Comment:
if we replace with readGcloudConfigRaw with a executeCommand, we should be able to add the tests reasonably easily as well as increase test coverage.